rds/variables.tf

```diff
@@ -127,3 +132,31 @@ variable "monitoring_interval" {
   description = "The interval, in seconds, between points when Enhanced Monitoring metrics are collected for the DB instance. To disable collecting Enhanced Monitoring metrics, specify 0. Valid Values: 0, 1, 5, 10, 15, 30, 60."
   default     = "0"
 }
+
+variable "default_db_parameters" {
```

Comment:
Maybe create this as a local too? Since now there's two ways for setting parameters:
- set the default_db_parameters var
- set the rds_custom_parameter_group_name var

Reply:
I didn't set it as a local because I thought we were overriding those parameters via the variable. But I can't find any example anywhere, so I'll change it to a local.

Reply:
done